django_fido/views.py

```diff
@@ -49,11 +53,13 @@ class Fido2ViewMixin(object):
                        see https://www.w3.org/TR/webauthn/#enumdef-attestationconveyancepreference
     @cvar attestation_types: Allowed attestation format types.
         If None, all attestation formats except `none` are allowed.
+    @cvar user_verification: The desired USER_VERIFICATION level.
     @cvar session_key: Session key where the FIDO 2 state is stored.
     """
 
     attestation = AttestationConveyancePreference.NONE
     attestation_types = None  # type: Optional[List[Attestation]]
+    user_verification = UserVerificationRequirement.PREFERRED if fido2.__version__ < '0.8' else None
```

Comment:
`PREFERRED` should work in older fido2 versions as well, shouldn't it?

Reply:
I thought maybe not to prescribe the default. It may be cleaner to set it, though. Tests have to be adapted in that case.

Reply:
I've checked the wrong version of `fido2` library. Sorry.